funcx_endpoint/funcx_endpoint/endpoint/manager.py

```diff
@@ -0,0 +1,343 @@
+import glob
+from importlib.machinery import SourceFileLoader
+import json
+import logging
+import os
+import pathlib
+import random
+import shutil
+import signal
+import sys
+import time
+import uuid
+from string import Template
+
+import daemon
+import daemon.pidfile
+import psutil
+import requests
+import texttable as tt
+import typer
+from retry import retry
+
+import funcx
+import zmq
+
+from funcx.utils.errors import *
+from funcx_endpoint.endpoint import default_config as endpoint_default_config
+from funcx_endpoint.executors.high_throughput import global_config as funcx_default_config
+from funcx_endpoint.endpoint.interchange import EndpointInterchange
+from funcx.sdk.client import FuncXClient
+
+
+class EndpointManager:
+    def __init__(self, logger):
+        self.FUNCX_CONFIG_FILE_NAME = 'config.py'
```

Comment:
please use [lowercase names](https://visualgit.readthedocs.io/en/latest/pages/naming_convention.html) for instance vars